Contrib/Inetc/inetc.cpp

```diff
@@ -1456,6 +1458,7 @@ INT_PTR CALLBACK dlgProc(HWND hDlg, UINT message, WPARAM wParam, LPARAM lParam )
     }
 	case WM_CTLCOLORDLG:
 	{
+		szBgColor[6] = 0;
```

Comment:
@Nutzzz As stated before, I'm really not up to speed with C++, but how you can set index 6 of ```szBgColor``` when if according to the next ```if``` might be ```NULL```? Won't that lead to an error if it is in fact ```NULL```?